test/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Should this be in a fixture directory instead?

Reply:
Then the project directory should be pointing to `fixtures/`, right?

Reply:
You’ll know better than me 😉 I just want to make sure it’s clear that it’s used with fixtures only.

Reply:
It's easier to keep this in `test` 😄